packages/vscode-extension/src/common/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { DocumentSelector } from 'vscode';
+
+/**
+ * The document selectors is the list of documents for which vscode needs to send requests/notifications to the language server.
+ *
+ * We specifically don't want to answer completion requests in package.json files and so on.
+ *
+ * Nor do we want to handle js,css,scss liquid files.
+ */
+export const documentSelectors: DocumentSelector = [
+  { language: 'liquid', pattern: '**/*.liquid' },
```

Comment:
We unfortunately can't do _negative_ glob patterns here. We rely on `language` to be set properly for the js/css/scss liquid files. 

Looking at packages/vscode-extension/package.json, it looks like I did a mapping for those files to a `language` that _isn't_ liquid. So I think we're good here.